### ISSUE TITLE
The mouse pointers are scaled improperly

### ISSUE THREAD

**Issue by davidcoleman007** [WhitewaterFoundry/Pengwin#267]
**Describe the bug**
first of all, I bought this yesterday and I love it.  it is a great distro and I see that a lot of great work has been done.  kudos.

However, I ran into something really annoying: The I-beam mouse pointerlooks bad.  It is cut off and renders larger than the pointer box area.  Also the click target is aligned somewhere along the bottom of the box (possibly out side of it).  it's difficult to use effectively.

(edit: I'm very aware that running gnome-terminal on wsl is a really bleeding edge and experimental thing, but I'd LOVE to see if there's a way we could fix the pointer and the combo boxes that don't show lists) :)

the same happens with vs code and the arrow pointer (actually, ALL pointers):
![image](https://user-images.githubusercontent.com/41075/50259238-1a341f00-03b8-11e9-879d-b43d6a0af476.png)

the scaling of the arrow pointer is correct in `gnome-terminal`, and also in `gnome-terminal`, the standard windows re-size pointers are used and work perfectly.

**To Reproduce**
I installed the app from the windows store, with X410, also with default settings from windows store.

the pointer hot point is also aligned with the bottom of the pointer box, or possibly with the baseline mark on the i-beam which is cut off and not rendered properly.

**Expected behavior**
the pointers look seamless with at least the standard windows ui, or properly scaled default ubuntu/xcfe/some other open system pointers.

a short term fix would be to reduce the scaling by about 50%, the resize pointer seems to show that they about double size:
![image](https://user-images.githubusercontent.com/41075/50259452-fc1aee80-03b8-11e9-8736-0c32f1cea5bc.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/41075/50257843-efdf6300-03b1-11e9-80e2-aa3d2e8d17cc.png)

**Additional context**
![image](https://user-images.githubusercontent.com/41075/50257969-6f6d3200-03b2-11e9-9151-32baad908ebb.png)

![image](https://user-images.githubusercontent.com/41075/50257975-7a27c700-03b2-11e9-8433-11a285c7266a.png)

**Basic Troubleshooting Checklist**

[x] I have searched Google for the error message.
[x] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[x] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[x] I have searched the WLinux issues page: https://github.com/WhitewaterFoundry/WLinux/issues.
[x] I have reset WLinux: Settings->Apps->Apps & features->WLinux->Advanced Options->Reset. 
[x] I have disabled and re-enabled WSL in Windows Features.
[x] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?

Insert here:

**WLinux Version**
1.1.27.0

Insert here:
```
:/mnt/f/var/code
$bash --version
GNU bash, version 4.4.23(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
:/mnt/f/var/code
$uname -a
Linux dave_laptop 4.4.0-17134-Microsoft #471-Microsoft Fri Dec 07 20:04:00 PST 2018 x86_64 GNU/Linux
:/mnt/f/var/code
$gnome-terminal --version
# GNOME Terminal 3.30.2 using VTE 0.54.2 +GNUTLS
:/mnt/f/var/code
$apt-cache show xserver-xorg | grep Version
Version: 1:7.7+19
```

**Windows Build**
buildLab: `17134.rs4_release.180410-1804`
currentBuild: `17134`

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:

```
C:\Users\david_000>systeminfo | findstr /C:"OS"
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
BIOS Version:              American Megatrends Inc. G751JY.205, 10/29/2014
```

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by crramirez** [comment]
Hello,

I have seen this behavior too. But this issue belongs to X410 which is the software which manages the graphical part.

Regards

**Comment by davidcoleman007** [comment]
:( are they open source? do they accept bugs? ;)  is there another x-server which works better?

**Comment by crramirez** [comment]
You can try VcXsrv which is open source. I don't know if X410 are open source.

**Comment by crramirez** [comment]
Update to the latest version of X410. I haven't seen this issue anymore

**Comment by crramirez** [comment]
Correction I still see the issue. If you have both monitors with highdpi and regular ones you will see big cursors

**Comment by marketideas** [comment]
Just a little more info on this as I just started seeing it, and was able to correct it based on the previous (crramirez) comment.
It probably only happens with multi-monitors. I typically have 3. 1920x1080, 1920x1080, and 1920x1200.  I recently added another monitor (4 total), but instead of side to side, I placed the 4th monitor above the middle monitor. It is also 1920x1080.  So now, my entire desktop is something like 5760x2160.  When in this mode, X410 scales the cursors to very large amounts. Disabling that 4th monitor, and restarting x410, all is well again.
